client/extensions/woocommerce/app/order/order-details/index.js

```diff
@@ -43,6 +43,13 @@ class OrderDetails extends Component {
 		}
 	};
 
+	updateLineItems = item => {
```

Comment:
Sorry - its a nit, but maybe this would be better named `updateLineItem` since it only accepts one item?

Reply:
It's actually changed to updateOrder in another PR (because it ends up taking line items and fees), so I'll leave as-is for now.